content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

```diff
@@ -226,7 +226,7 @@ To deploy the W&B platform, the Kubernetes Operator uses the values from your CR
 
 Replace all tags/versions with the versions that are available in your internal registry.
 
-More information on creating the preceding configuration file can be found [here]({{< relref "/guides/hosting/hosting-options/self-managed/kubernetes-operator/#configuration-reference-for-wb-server" >}}). 
+For more information, see the [W&B Operator Configuration reference]({{< relref "/guides/hosting/hosting-options/self-managed/kubernetes-operator/#configuration-reference-for-wb-server" >}}) and the [example `wandb.yaml` file](https://github.com/wandb/helm-charts/blob/main/charts/operator/crds/wandb.yaml) in the Helm chart repository.
```

Comment:
This isn't an example of `wandb.yaml` we want customers to use, but the CRD that we install when we deploy the W&B operator.

The rule of thumb here, I'd say, is never suggest to customers that they modify the CRD.

```
and the [example `wandb.yaml` file](https://github.com/wandb/helm-charts/blob/main/charts/operator/crds/wandb.yaml) in the Helm chart repository.
```

What we want from customers is to create CR instances from our CRD.

Reply:
Makes sense, I can remove it, but I wonder why we had it in the first place.